### ISSUE TITLE
Unexpected URL-encoding of path in ReplacePathRegex middleware

### ISSUE THREAD

**Issue by ableuler** [containous/traefik#6355]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I tried to URL-encode a `/` in a path using the ReplacePathRegex middleware. Traefik took the `%` sign of `%2F` and encoded it again, resulting in `%252F` instead.
Is this related to this middleware handling the `Path` rather than the `RawPath` property, similar to the problem that was fixed in https://github.com/containous/traefik/pull/2382/files?

### What did you expect to see?
e.g.
```
/entities/projects/some-namespace/some-project 
--> 
/entities/projects/some-namespace%2Fsome-project
```

### What did you see instead?

```
/entities/projects/some-namespace/some-project 
--> 
/entities/projects/some-namespace%252Fsome-project
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.1.4
Codename:     cantal
Go version:   go1.13.7
Built:        2020-02-06T17:10:06Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
[http.middlewares.urlEncodeForwardSlash.ReplacePathRegex]
  regex = "/entities/([^/]*)/([^/]*)/(.*)"
  replacement = "/entities/${1}/${2}%2F${3}"
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
time="2020-02-22T14:58:15Z" level=debug msg="vulcand/oxy/roundrobin/rr: completed ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/entities/projects/andreas%2Fmy-project\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"application/json\"],\"Accept-Encoding\":[\"gzip, deflate\"],\"Cache-Control\":[\"no-cache\"],\"Cookie\":[\"session=9b14534af647ce28_5e4ef2c3.RaUgKduhfJFR-vq-sjl-4-8Wsyo; sails.sid=s%3ASKKV5IEbjSWTk_g_se8t3HjhqSwFhCxx.j0kFSx%2Fr8ogYMR5T8GyroHEzNbPGmO70mv29rbVj1Dc\"],\"User-Agent\":[\"PostmanRuntime/7.20.1\"],\"X-Forwarded-Host\":[\"andreas.dev.renku.ch\"],\"X-Forwarded-Port\":[\"80\"],\"X-Forwarded-Proto\":[\"http\"],\"X-Forwarded-Server\":[\"andreas-renku-gateway-7c96dc49c9-nfgrz\"],\"X-Original-Uri\":[\"/entities/projects/andreas/my-project\"],\"X-Real-Ip\":[\"10.42.3.61\"],\"X-Replaced-Path\":[\"/entities/projects/andreas/my-project\"],\"X-Request-Id\":[\"be0c6c922b80b103dd14f49871bd9717\"],\"X-Requested-With\":[\"XMLHttpRequest\"],\"X-Scheme\":[\"https\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"andreas.dev.renku.ch\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"10.42.3.61:33898\",\"RequestURI\":\"/entities/projects/andreas%252Fmy-project\",\"TLS\":null}"
```

**Comment by traefiker** [comment]
Closed by #6413.

**Comment by ableuler** [comment]
Thanks to everybody involved for the quick action on this issue 👍!
